### ISSUE TITLE
ERROR: Can't load the input model: Data loss: Can't parse 

### ISSUE THREAD

**Issue by sheeceg** [remicres/sr4rs#64]
I am unsure why this is happening I remember testing it out a few months ago and it preformed really good. Any pointer will be greatly appreciated. 

```
otbuser@836a81181195:~$ python sr4rs/code/sr.py \
> --savedmodel sr4rs_sentinel2_bands4328_france2020_savedmodel \
> --input test_1_august_163558_4328.tif \
> --output test.tif
2023-03-26 03:24:10 (INFO) ReadImageInfo: 
Image general information:
	Number of bands : 4
	Data type : unsigned_short
	No data flags : Not found
	Start index :  [0,0]
	Size :  [256,256]
	Origin :  [1.59663e+06,-2.54157e+06]
	Spacing :  [9.99977,-10.003]
	Estimated ground spacing (in meters): [10.0918,9.95377]

Image acquisition information:
	Sensor : 
	Image identification number: 
	Image projection : PROJCS["GDA94 / Australian Albers",
    GEOGCS["GDA94",
        DATUM["Geocentric_Datum_of_Australia_1994",
            SPHEROID["GRS 1980",6378137,298.257222101,
                AUTHORITY["EPSG","7019"]],
            TOWGS84[0,0,0,0,0,0,0],
            AUTHORITY["EPSG","6283"]],
        PRIMEM["Greenwich",0,
            AUTHORITY["EPSG","8901"]],
        UNIT["degree",0.0174532925199433,
            AUTHORITY["EPSG","9122"]],
        AUTHORITY["EPSG","4283"]],
    PROJECTION["Albers_Conic_Equal_Area"],
    PARAMETER["latitude_of_center",0],
    PARAMETER["longitude_of_center",132],
    PARAMETER["standard_parallel_1",-18],
    PARAMETER["standard_parallel_2",-36],
    PARAMETER["false_easting",0],
    PARAMETER["false_northing",0],
    UNIT["metre",1,
        AUTHORITY["EPSG","9001"]],
    AXIS["Easting",EAST],
    AXIS["Northing",NORTH],
    AUTHORITY["EPSG","3577"]]

Image default RGB composition:
	[R, G, B] = [0,1,2]

Ground control points information:
	Number of GCPs = 0
	GCPs projection = 

2023-03-26 03:24:10 (INFO) TensorflowModelServe: Default RAM limit for OTB is 256 MB
2023-03-26 03:24:10 (INFO) TensorflowModelServe: GDAL maximum cache size is 6424 MB
2023-03-26 03:24:10 (INFO) TensorflowModelServe: OTB will use at most 24 threads
2023-03-26 03:24:10.782082: I tensorflow/cc/saved_model/reader.cc:38] Reading SavedModel from: sr4rs_sentinel2_bands4328_france2020_savedmodel
2023-03-26 03:24:10.862135: I tensorflow/cc/saved_model/loader.cc:277] SavedModel load for tags { serve }; Status: fail: Data loss: Can't parse sr4rs_sentinel2_bands4328_france2020_savedmodel/saved_model.pb as binary proto. Took 80055 microseconds.
Traceback (most recent call last):
  File "sr4rs/code/sr.py", line 98, in <module>
    infer.ExecuteAndWriteOutput()
  File "/opt/otbtf/lib/otb/python/otbApplication.py", line 2321, in ExecuteAndWriteOutput
    return _otbApplication.Application_ExecuteAndWriteOutput(self)
RuntimeError: Exception thrown in otbApplication Application_ExecuteAndWriteOutput: /src/otb/otb/Modules/Remote/otbtf/include/otbTensorflowGraphOperations.cxx:61:
itk::ERROR: Can't load the input model: Data loss: Can't parse sr4rs_sentinel2_bands4328_france2020_savedmodel/saved_model.pb as binary proto

```

**Comment by sheeceg** [comment]
I get same error using 

```
docker run -it --rm -v $(pwd):/home/otbuser --runtime=nvidia mdl4eo/otbtf2.5:cpu bash
docker run -it --rm -v $(pwd):/home/otbuser --runtime=nvidia mdl4eo/otbtf2.5:gpu bash
```

I am on Ubuntu and have NVIDIA GPU.

**Comment by remicres** [comment]
hi @sheeceg ,

Could you try with a more recent docker image?
Not sure this will fix your issue, but 2.5 is really old.

Also, try to download again the model. Your files looks corrupted, or maybe it is a file permission issue...

**Comment by sheeceg** [comment]
I think its related to file permission issue, when I removed "-v $(pwd):/home/otbuser" option it just worked!
